packages/cozy-harvest/src/Manifest.js

```diff
@@ -0,0 +1,63 @@
+export const legacyLoginFields = [
+  'login',
+  'identifier',
+  'new_identifier',
+  'email'
+]
+
+export const roleShouldBeRequired = ['identifier', 'password']
+
+export const legacyEncryptedFields = [
+  'secret',
+  'dob',
+  'code',
+  'answer',
+  'access_token',
+  'refresh_token',
+  'appSecret'
+]
+
+export const Manifest = {
+  sanitize: manifest => {
+    if (!manifest.fields) return manifest
+    manifest = Manifest.sanitizeIdentifier(manifest)
```

Comment:
As a good practice we should not mutate the manifest passed as argument and clone it instead : 
```js
const sanitized = { ...manifest }
sanitized.fields = sanitizedFields(sanitized.fields)
// ...

const sanitizedFields = fields => {
  const sanitized = { ...fields }
  sanitized = sanitizeIdentifier(sanitized)
  return sanitized
}

// etc...
```

With this pattern we can prevent any mutation on the original manifest parameter.